### DISCUSSION TITLE
webdav 修改的或者创建的部分小文件 损坏，重新修改后，云盘版本正常但是webdav 访问不正常或者未修改。

### DISCUSSION THREAD

**Discussion by MelanPrince** [messense/aliyundrive-webdav#418] [Q&A] [answered]
### 问题描述

小碎片图片文件损坏。

### 重现步骤

    阿里云电影使用 rclone 和 RaiDrive 挂载本地盘（起初使用rclone我以为是Rclone的问题，更换RaiDrive 依旧有问题）

    使用 tinyMediaManager 4 刮削电影，从阿里云的删除记录来看，他应该是先删除原来刮削的nfo和图片，然后重新下载一份。刮削完成后 有很多图片只有半幅图片，重新修改后有些可以改正有些依旧是错误。

    从网页网文webdav 发现和阿里云盘客户端文件修改日期不一致。阿里云盘可以是最新修改的并且是没有损坏的，但是webdav 访问的就是损坏的旧的日期，云盘删除 webdav 还有，webdav 删除 云盘删除。

### 版本

梅林1.3.1 和docker 版本都是

### 运行平台

AX86U和 UNRAID

### 日志

https://pic.imgdb.cn/item/6274951d0947543129b1626f.png

https://pic.imgdb.cn/item/627495310947543129b196f7.png

**Comment by MelanPrince** [comment]
试验使用新的 refresh token 发现一样没有改变

另外 docker 版本的ROOT 设置根目录 无效。

**Comment by messense** [comment, marked as answer]
Duplicate of https://github.com/messense/aliyundrive-webdav/discussions/372
